Source/WebCore/platform/graphics/gstreamer/GRefPtrGStreamer.cpp

```diff
@@ -28,6 +28,7 @@
 #endif
 
 #if USE(GSTREAMER_WEBRTC)
+#include <gst/rtp/gstrtphdrext.h>
```

Comment:
Why not just <gst/rtp/rtp.h> ?

Reply:
That's what LSP inserted for me and I didn't complain :) We don't need the full and big rtp includes here after all.

Reply:
AFAIK the grouping headers are preferred by GStreamer, I guess because possible changes in the future. But perhaps it's just bikesheding :)

Reply:
let's do the right thing then!